gradle/libs.versions.toml

```diff
@@ -0,0 +1,231 @@
+[versions]
+agp = '8.1.0'
+androidInstallReferrer = '2.2'
+androidxActivity = '1.8.0'
+androidxAppcompat = '1.6.1'
+androidxArchCore = '2.2.0'
+androidxCamera = '1.3.4'
+androidxCardview = '1.0.0'
+androidxComposeMaterial3 = '1.1.1'
+androidxComposeNavigation = '2.7.6'
+androidxConstraintlayout = '2.1.4'
+androidxConstraintlayoutCompose = '1.0.1'
+androidxCore = '1.10.0'
+androidxFragment = '1.6.2'
+androidxLifecycle = '2.6.2'
+androidxPercentLayout = '1.0.0'
+androidxPreference = '1.2.0'
+androidxRecyclerview = '1.3.0'
+androidxSwipeRefreshLayout = '1.1.0'
+androidxTest = '1.5.0'
+androidxTestEspresso = '3.4.0'
+androidxTestExtJunit = '1.1.5'
+androidxTestUiAutomator = '2.2.0'
+androidxViewpager2 = '1.0.0'
+androidxWebkit =  '1.11.0'
+androidxWorkManager = "2.9.0"
+apacheCommonsText = '1.10.0'
+assertj = '3.23.1'
+automatticRest = '1.0.8'
+cascadeCompose = '2.3.0'
+chrisbanesPhotoview = '2.3.0'
+coilCompose = '2.4.0'
+composeBom = '2023.10.00'
+dagger = '2.50'
+dependencyAnalysis = "1.28.0"
+detekt = "1.23.0"
+eventBus = '3.3.1'
+exoPlayer = '2.13.3'
+facebookReact = '0.73.3'
+facebookShimmer = '0.5.0'
+firebaseBom = '32.1.0'
+firebaseIid = '21.1.0'
+flipper = '0.245.0'
+glide = '4.16.0'
+googleFlexbox = '3.0.0'
+googleMaterial = '1.9.0'
+googleMLKitBarcodeScanning = '17.2.0'
+googleMLKitBarcodeScanningCommon = '17.0.0'
+googleMLKitTextRecognition = '16.0.0'
+googlePlayAppUpdate = '2.1.0'
+googlePlayReview = '2.0.1'
+googlePlayServicesAuth = '20.4.1'
+googleServices = '4.3.15'
+gson = '2.10.1'
+indexosMediaForMobile = '43a9026f0973a2f0a74fa813132f6a16f7499c3a'
+jsoup = '1.16.2'
+junit = '4.13.2'
+kotlin = '1.9.22'
+kotlinxCoroutines = '1.7.3'
+kover = "0.7.5"
+ksp = "1.9.22-1.0.17"
+lottie = '6.1.0'
+measureBuilds = '2.1.2'
+mockitoAndroid = '4.5.1'
+mockitoKotlin = '4.1.0'
+navigation = "2.7.7"
+philjayMpAndroidChart = 'v3.1.0'
+screengrab = '2.1.1'
+sentry = "4.3.1"
+soLoader = '0.10.5'
+squareupKotlinPoet = '1.16.0'
+squareupMockWebServer = '4.12.0'
+squareupRetrofit = '2.9.0'
+uCrop = '2.2.9'
+violationComments = '1.70.0'
+volley = '1.2.1'
+wiremock = '2.26.3'
+wiremockHttpClient = '4.3.5.1'
+wordPressPersistentEditText = '1.0.2'
+zendesk = '5.1.2'
+
+[libraries]
+android-appUpdate = { group = "com.google.android.play", name = "app-update", version.ref = "googlePlayAppUpdate" }
+android-appUpdateKtx = { group = "com.google.android.play", name = "app-update-ktx", version.ref = "googlePlayAppUpdate" }
```

Comment:
Minor (🔍): Just like you did everywhere else, please update this dependency to `android-appUpdate-ktx` instead.

Reply:
fixed.